lib/phoenix_live_view/tokenizer.ex

```diff
@@ -1,10 +1,59 @@
-defmodule Phoenix.LiveView.HTMLTokenizer do
+defmodule Phoenix.LiveView.Tokenizer do
   @moduledoc false
   @space_chars '\s\t\f'
   @quote_chars '"\''
   @stop_chars '>/=\r\n' ++ @quote_chars ++ @space_chars
 
-  alias __MODULE__.ParseError
+  alias __MODULE__.{ParseError, HTML}
+
+  defmodule TagHandler do
```

Comment:
Let's define this behaviour at `Phoenix.LiveView.TagEngine.Tokenizer` or `Phoenix.LiveView.TagHandler`. I want to avoid defining a new namespace. You can define the HTML tag handler at `Phoenix.LiveView.HTMLEngine` itself! :)